### PR TITLE
feature: add folder structure

### PR DESCRIPTION
**Pull request by samdev254**
This PR currently contains the **folder structure** of the entire application

**Note:** please note that most of the files created are empty and some modules aren't exported as well, once we refactor everything to actual components then we can modularise and export them as required. Thanks 

🙋‍♂️ _"Feel free to raise any issue or concerns"_

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,12347 +1,8 @@
 {
   "name": "droidconke2022reactnative",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (brianwachira) [inline package-lock.json:4]: Im seeing lockfileversion has changed to lofckfileVersion1. Should this be a concern?

**Review by brianwachira — CHANGES_REQUESTED** [review]

**Comment by makunomark** [comment]
might I request for a documentation of the folder structure?

File: package-lock.json (hunk 1 above)
Comment (wafulasam) [reply package-lock.json:4]: I think it's something to do with npm versions. lockfileversion 2 is NPM 7 while 6 is lockfileversion 1. Not sure just a thought though 🤔

**Comment by wafulasam** [comment]
@makunomark  yeah sure, will sort out https://github.com/droidconKE/droidconKE2022ReactNative/pull/3#issuecomment-1238576527 currently in progress. 

At first I thought it wasn't a necessity since things are already laid out, just building on top of what has already been setup

File: package-lock.json (hunk 1 above)
Comment (brianwachira) [reply package-lock.json:4]: > I think it's something to do with npm versions. lockfileversion 2 is NPM 7 while 6 is lockfileversion 1. Not sure just a thought though thinking

@makunomark @orama254 what do you think about this? personally I think that if it won't break the project then that's okay.

**Review by brianwachira — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
-    "@types/react": "~18.0.14",
+    "@types/react": "^18.0.18",
```
Comment (brianwachira) [inline package.json:21]: types/react version has changed to 18.0.18
Will package versions be changing every time someone npm installs?

**Review by orama254 — APPROVED** [review]

**Comment by makunomark** [comment]
> @makunomark yeah sure, will sort out [#3 (comment)](https://github.com/droidconKE/droidconKE2022ReactNative/pull/3#issuecomment-1238576527) currently in progress.
> 
> At first I thought it wasn't a necessity since things are already laid out, just building on top of what has already been setup

I agree. There's a chance as we proceed it will take shape.  cc @wafulasam

**Review by makunomark — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (makunomark) [reply package-lock.json:4]: > I think it's something to do with npm versions. lockfileversion 2 is NPM 7 while 6 is lockfileversion 1. Not sure just a thought though 🤔

I thought so too. We locked npm version to those above 7.0.0 in package.json. Didn't seem to enforce. 

@wafulasam Which version are you using locally?

Comment (brianwachira) [reply package-lock.json:4]: > > I think it's something to do with npm versions. lockfileversion 2 is NPM 7 while 6 is lockfileversion 1. Not sure just a thought though thinking
> 
> @makunomark @orama254 what do you think about this? personally I think that if it won't break the project then that's okay.

**UPDATE**
- For now, let's merge this PR, then create an issue to explore whether lock file versions changing will break the project (somehow)

**Pull request merged**